plugins/woocommerce/tests/e2e-pw/tests/shopper/cart-block-calculate-shipping.spec.js

```diff
@@ -166,12 +175,21 @@ test.describe( 'Cart Block Calculate Shipping', () => {
 		await addAProductToCart( page, product1Id );
 		await page.goto( cartBlockPage.slug );
 
+		await page
+			.locator( 'button', {
+				hasText: 'Enter the address to check delivery options',
+			} )
+			.click();
+
 		// Set shipping country to Portugal
-		await page.getByLabel( 'Add an address for shipping' ).click();
 		await page.getByRole( 'combobox' ).first().fill( 'Portugal' );
 		await page.getByLabel( 'Postal code' ).fill( '1000-001' );
 		await page.getByLabel( 'City' ).fill( 'Lisbon' );
-		await page.getByRole( 'button', { name: 'Update' } ).click();
+		await page
+			.locator(
+				'.wc-block-components-shipping-calculator-address__button'
+			)
+			.click();
```

Comment:
See my previous comment regarding using `page.getByRole()` instead of `page.locator()`.